src/compiletest/errors.rs

```diff
@@ -84,8 +124,9 @@ fn parse_expected(last_nonfollow_error: Option<usize>,
     let letters = line[kind_start..].chars();
     let kind = letters.skip_while(|c| c.is_whitespace())
                       .take_while(|c| !c.is_whitespace())
```

Comment:
Addressed in https://github.com/rust-lang/rust/pull/32341/commits/73f432189298dd07195427fcc9216ed2c5ba8d11